vendor/vendor.json

```diff
@@ -616,6 +616,13 @@
 			"revision": "7d9177d70076375b9a59c8fde23d52d9c4a7ecd5",
 			"revisionTime": "2017-09-15T19:08:28Z"
 		},
+		{
+			"checksumSHA1": "6MJzEG/bwAT3Ljra9h+l7UR3il8=",
+			"origin": "github.com/minio/minio/vendor/golang.org/x/crypto/ocsp",
```

Comment:
this vendor looks wrong @vadmeste it should be `golang.org/x/crypto/ocsp`

Reply:
> this vendor looks wrong @vadmeste it should be golang.org/x/crypto/ocsp

Fixed